conans/test/unittests/client/generators/virtualrunenv_test.py

```diff
@@ -20,13 +23,14 @@ def setUpClass(cls):
         conanfile.deps_cpp_info["hello"].lib_paths = ["lib1", "lib2"]
 
         cls.generator = VirtualRunEnvGenerator(conanfile)
+        cls.generator.output_path = "not-used"
```

Comment:
Why is it necessary to define the output_path everywhere now?

Reply:
The `output_path` is always provided when running Conan, it's in the tests where we are not setting it. 

We need it now because I'm storing the full path to the generated `environment.XXX.env` file ([here for example](https://github.com/conan-io/conan/pull/5989/files#diff-27295679c78347c910d20b9e59c47849R185)). It is safer to have the fullpath there than to compute the path relative to the `.sh`, `.cmd`, `.ps1` scripts to find the `.env` file.